spec/requests/linking/link_defendant_no_maat_id_spec.rb

```diff
@@ -9,6 +9,13 @@
   let(:defendant_id) { defendant_id_from_fixture }
   let(:defendant_id_from_fixture) { '41fcb1cd-516e-438e-887a-5987d92ef90f' }
   let(:commit) { 'Create link without MAAT ID' }
+  let(:maat_error_message) do
```

Comment:
This would be better defined within the test context block since it is not reused elsewhere

Reply:
Should be resolved with most recent push